silkworm/rpc/core/storage_walker.cpp

```diff
@@ -177,46 +177,43 @@ Task<void> StorageWalker::walk_of_storages(
     co_return;
 }
 
+static void increment_key(Bytes& array) {
+    for (auto& it : std::ranges::reverse_view(array)) {
+        if (it < 0xFF) {
+            ++it;
+            break;
+        }
+        it = 0x00;
+    }
+}
```

Comment:
This exact function is present also in `parity_api.cpp`, we should move it to `rpc/common` folder. You can do this in next PR